R/package-info.R

```diff
@@ -138,7 +138,11 @@ pkg_source <- function(desc) {
     repo
 
   } else if (!is.null(desc$biocViews)) {
-    "Bioconductor"
+    if (desc$biocViews != "") {
+      "Bioconductor"
+    } else {
+      "local"
+    }
```

Comment:
It is better to write this as 
```r
} else if (!is.null(desc$biocViews) && desc$biocViews != "") {
```
and drop the `else`.  It is better to avoid nested ifs, in case more branches will be added at the end later.